src/ui/handlers/run-info-ui-handler.ts

```diff
@@ -819,10 +819,11 @@ export class RunInfoUiHandler extends UiHandler {
         moveContainer.add(moveLabel);
         movesetContainer.add(moveContainer);
         const move = pokemonMoveset[m]?.getMove();
+        const moveName = move ? pokemon.getMove(move.id)!.name : "-";
         pokemonMoveBgs[m].setFrame(
           enumValueToKey(ElementalType, move ? move.type : ElementalType.UNKNOWN).toLowerCase(),
         );
```

Comment:
```suggestion
        const pokemonMove = pokemonMoveset[m];
        const moveName = pokemonMove?.name ?? "-";
        const moveType = pokemonMove?.getMove()?.type ?? ElementalType.UNKNOWN;
        pokemonMoveBgs[m].setFrame(enumValueToKey(ElementalType, moveType).toLowerCase());
```
Trying to make more sense of this